### PR TITLE
redis: how to troubleshoot connection issues

### PR DESCRIPTION
**Pull request by allypascoe-aiven**
# What changed, and why it matters
Adding page for troubleshooting Redis connection issues

### CODE REVIEW

File: docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Discover troubleshooting techniques for your Redis®* service and check out how to resolve common connection issues.
+
+Some Redis®* connections are closed intermittently 
+--------------------------------------------------
+When experiencing connection issues with your Redis®* service, here are some common things to check:
+
+- By default :doc:`Aiven for Redis® uses SSL connecions</docs/products/redis/howto/manage-ssl-connectivity>` and as such, these connections are closed automatically after 12 hours. This is not a configurable parameter so it cannot be changed.
+- Some Redis®* clients do not support SSL connections. It is recommended to check the documentation for the Redis®* client being used to ensure SSL connections are supported.
+- If you are noticing older connections terminating, you should check to see what the value is configured for the :doc:`redis_timeout advanced parameter</docs/products/redis/reference/advanced-params>`. This parameter controls the timeout value for idle connections. Once the timeout is reached, the connection is terminated.
```
Comment (lionbee) [inline docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst:12]: Can we list this scenario first and indicate that by default Aiven does add a timeout?

File: docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Troubleshoot Redis®* connection issues
+######################################
+
+Discover troubleshooting techniques for your Redis®* service and check out how to resolve common connection issues.
+
+Important notes
+---------------
+By default :doc:`Aiven for Redis® uses SSL connecions</docs/products/redis/howto/manage-ssl-connectivity>` and as such, these connections are closed automatically after 12 hours. This is not a configurable parameter so it cannot be changed. Aiven also sets a default value of 300 seconds for the redis_timeout advanced parameter.
```
Comment (bridgetb) [inline docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst:8]: Is that `connecions` a typo? Or part of the link?

Comment (allypascoe-aiven) [reply docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst:8]: Good catch, I thought I fixed this up previously :grimacing: thank you :grin:

File: docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Troubleshoot Redis®* connection issues
+######################################
+
+Discover troubleshooting techniques for your Redis®* service and check out how to resolve common connection issues.
+
+Important notes
+---------------
+By default :doc:`Aiven for Redis® uses SSL connections</docs/products/redis/howto/manage-ssl-connectivity>` and as such, these connections are closed automatically after 12 hours. This is not a configurable parameter so it cannot be changed. Aiven also sets a default value of 300 seconds for the redis_timeout advanced parameter.
```
[the remark this answers is not in the document]
Comment (allypascoe-aiven) [reply docs/products/redis/troubleshooting/troubleshoot-redis-connection-issues.rst]: Thank you Ed :D

**Review by bridgetb — APPROVED** [review]
LGTM

**Pull request merged**